### ISSUE TITLE
[FEATURE] PLEASEE add these two features...

### ISSUE THREAD

**Issue by Sheikhm007** [Khiro95/Awqat-Salaat#31]
First of all, this app is already better than what I expected.
Thank you for making this app and keeping it free.

I would want 2 simple additions to this app.
First one would be an option to play the ADHAN when the prayer waqt (time) begins.

Secondly I would love to have the option to play a surah or custom file when my pc starts. Could you add the option to play a audio file on startup please.

Thanks again,
Sheikh.

**Comment by Khiro95** [comment]
Salam @Sheikhm007,

Thank you for the kind words and I'm happy that you liked the widget :)

Firstly, the request to add Adhan when the prayer time enters is legitimate and can be implemented. However, I'm afraid that the other request is out of the scope of this widget and will not be fulfilled. You can already achieve your goal using **Task Scheduler** on Windows.

Secondly, I'm working at the moment on finishing the updates for the next release (v3.3) and I may have to postpone your request to the later release (most probably will be v4).

Let me know if it's urgent for you to have Adhan feature or it's okay to wait a little.

Thanks for showing interest in the widget,
Khiro

**Comment by Sheikhm007** [comment]
Yeah sure, I would love to have the Adhan feature, it's completely okay if I need to wait for v4 to get that feature. And thank you for taking my suggestion.

**Comment by Khiro95** [comment]
Salam @Sheikhm007, 

v4 is out now with adhan added :)
